lib/membership/member.js

```diff
@@ -250,4 +184,46 @@ Member.statusPrecedence = function statusPrecedence(status) {
     }
 };
 
+/**
+ * Determine if the incoming change should be processed or not.
+ * The update should be processed if:
+ *  - the update is about an unknown member and the status is not 'tombstone'
+ *  - the new status takes precedence over the current known status
+ *    (@see Member~statusPrecedence)
+ *  - the incarnation number is newer
+
+ * @param {Member} [member] The current member or null when the member is currently
+ *  unknown.
+ * @param {object} change The incoming update
```

Comment:
can we use a more precise definition for a change than just a plane object? Wasn't there an `IMember` or will this be added in a later PR?